README.md

```diff
@@ -1 +1,63 @@
 # Tiidelab-Cohort-4
+> A personal professional portfolio of Victor Ade-Samuel
+
+
+## Built With
+
+- HTML5
+- CSS3
+-Javacsript
+
+
+## Live Demo 
+
+[Live Demo Link](https://victoradesamuel.github.io/Tiidelab-Cohort-4/)
+
+
+
+### Prerequisites
+- Good knowledge of Figma
+- Create Github repository
+- Have VSCode or choice text editor installed
+- Have Git installed
+- HTML/CSS Linters
+
+### Setup
+- Git clone - https://github.com/victoradesamuel/Tiidelab-Cohort-4
```

Comment:
- [ ] I think it would be more informative if you add all steps of installing and using your app. For example, using `git clone <repo link>`,  `cd <repo name>`, and `opening the app in the live server` steps can be added.